core_backend/tests/rails/test_safety.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from core_backend.app.configs.llm_prompts import SafetyClassification
+from core_backend.app.llm_call.parse_input import classify_safety
+
+pytestmark = pytest.mark.rails
+
+
+PROMPT_INJECTION_FILE = "tests/rails/data/prompt_injection_data.txt"
+SAFE_MESSAGES_FILE = "tests/rails/data/safe_data.txt"
+INAPPROPRIATE_LANGUAGE_FILE = "tests/rails/data/inappropriate_lang.txt"
```

Comment:
Similarly had to prepend "core_backend/" to the paths for the tests to find the files correctly

Reply:
Weird. Maybe because you are running it from the root? Mind trying `pytest -m rails` from core_backend?

Reply:
It passes for me! Probably gpt being weird :/ We should use a better model for this in the future.